packages/flutter/test/material/banner_theme_test.dart

```diff
@@ -67,11 +67,11 @@ void main() {
   });
 
   testWidgets('Passing no MaterialBannerThemeData returns defaults', (WidgetTester tester) async {
-    final ThemeData theme = ThemeData();
+    final ThemeData theme = ThemeData(useMaterial3: true);
```

Comment:
Very odd that this was still being used for some of the test values when it was an M2 theme, but the test was using M3. Nice fix.